mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -216,3 +216,38 @@ Feature: Loans tests
     Then status 200
     Then match $.loans[0].dueDateChangedByRecall == true
     And match $.loans[0].dueDate !contains expectedDueDateBeforeRequest
+
+  Scenario: When an loaned item is checked in at a service point that serves its location and no request exists, change the item status to Available
+
+    * def extItemBarcode = '' + random(100000)
+    * def extUserId = call uuid1
+    * def extUserBarcode = random(1000000)
+
+    # post an item
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostServicePoint') { id: #(servicePointId) }
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostHoldings')
+    * call read('classpath:domain/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(itemId), extItemBarcode: #(extItemBarcode)}
```

Comment:
Can we reorder these or group them differently? Locations and service points are related to the institution structure but instances, holdings and items are more about inventory.